app/BeaconBridge/Constants/Submission/EgressStatus.cs

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright [yyyy] [name of copyright owner]
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
```

Comment:
Do you need to leave this here?

Reply:
It's a model from the DARE-Control repo, and they have apache license, so I need to add to the files according to the license. But they didn't apply it properly so they have no copyright owner listed, so that's I left the default one

Reply:
Moved them to their own class instead